src/Tutor.Infrastructure/Database/Repositories/LearningObjectDatabaseRepository.cs

```diff
@@ -43,7 +43,7 @@ public Challenge GetChallenge(int challengeId)
                 .Where(c => c.Id == challengeId)
                 .Include(c => c.Solution)
                 .Include(c => c.FulfillmentStrategies)
-                    .ThenInclude(s => (s as BasicMetricChecker).ClassMetricRules)
+                    .ThenInclude(s => (s as BasicMetricChecker).MetricRanges)
                     .ThenInclude(r => r.Hint)
                 .Include(c => c.FulfillmentStrategies)
                     .ThenInclude(s => (s as BasicMetricChecker).MethodMetricRules)
```

Comment:
**Important**: Update the model, migrations, and scripts to reflect that we now have a single set of metric rules.